projects/angular-codegen/src/lib/angular-bootstrap.service.ts

```diff
@@ -0,0 +1,100 @@
+import { Injectable } from '@angular/core';
+import { FormatService } from '@xlayers/sketch-lib';
+
+type WebCodeGenOptions = any;
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AngularBootstrapService {
```

Comment:
Can you explain why you introduce the concept of a bootstrap service? I believe it will help in decoupling the templating from the original `codegen` services.   If this is the main reason do we agree that we should introduce this concept also for  the other generators?